app/src/app/navtitle/navtitle.component.sass

```diff
@@ -35,6 +37,11 @@ h2
   position: relative
   top: 3px
 
+  @media (max-width: $breakpoint-tablet)
+    top: -3px
+    font-size: small
```

Comment:
Absolute symbolic size are bad, because they may be different on different browsers. The font size is hard-coded on line 33. Hard-code it here too.